.github/workflows/macos.yaml

```diff
@@ -16,33 +16,58 @@ jobs:
       matrix:
         ghc: ["9.6.4"]
         cabal: ["3.10"]
-        os: ["macOS-latest"]
```

Comment:
we should do both - one goes through the macos github runner, but one is fixed hardware we have onhand for M1 specifically for regression.

Reply:
Checked and Macos-latest is macos-14